app/src/main/res/layout/view_watchlist_main_tooltip.xml

```diff
@@ -5,6 +5,7 @@
     android:orientation="vertical"
     android:paddingStart="16dp"
     android:paddingTop="16dp"
+    android:paddingBottom="16dp"
```

Comment:
Adding `16dp` looks will show too much space on the bottom (on Samsung S9). Would it be possible to change it to `8dp`?

Reply:
Here's how it looks on my Pixel 2:

![image](https://user-images.githubusercontent.com/1682214/110020620-ce02fa00-7cf7-11eb-952c-2a632143ed7b.png)

Are we sure the Balloon updates are ready for prime time?

Reply:
I see what's happening there. 
The library uses the `width` instead of `height` to adjust the tooltip's height.

My Pixel 3 XL also has a similar issue you have on Pixel 2 (with `android:paddingBottom="16dp"` changes)

It would be better to change `android:padding='16dp'` for a proper fix.

Reply:
Should we add the `ScrollView` back to the tooltip custom views? I will report this issue to the author. 🆘